### PR TITLE
Add 403 dataset_too_large_for_offset_based_paging for Trash API to spec

### PR DESCRIPTION
**Pull request by cbetta**
Also adds a changelog notification for DDOC 246 and 247.

### CODE REVIEW

File: content/pages/changelog/index.md [hunk 1]
```diff
@@ -10,6 +10,23 @@ rank: 0
 For historical changelog entries, please see
 [2018 Release Notes](page://changelog/2018).
 
+## 2019-11-25 / Marker-based Pagination Support for Trash API
```
Comment (jcleblanc) [inline content/pages/changelog/index.md]: Can you pull in the current updates to the changelog and merge please? There is a conflict with the launch of the Elements 11 changelog entry.

Comment (cbetta) [reply content/pages/changelog/index.md]: Done

File: content/pages/changelog/index.md [hunk 2]
```diff
@@ -10,6 +10,23 @@ rank: 0
...
+
+The [Trash API](e://get_folders_trash_item) now support [marker-based
```
Comment (elanelango) [inline content/pages/changelog/index.md]: now supports* ?

File: content/pages/changelog/index.md [hunk 3]
```diff
@@ -10,6 +10,23 @@ rank: 0
...
+pagination](g://api-calls/pagination/marker-based). Marker-based pagination can
+be used to access any user's trash and is the required pagination option for
+users with more than 100,000 items in their trash.
+
+## 2019-11-25 / New Trash Scale-Limit Error Code
+
+The [Trash API](e://get_folders_trash_items) will now return a new `403
+Forbidden` error when accessing very large trash folders. This error resolves an
+issue where trash folders have become unresponsive for users with more than
+100,00 items in their trash folder.
```
Comment (elanelango) [inline content/pages/changelog/index.md]: 100,000 - missing a zero.

File: package.json [hunk 4]
```diff
@@ -19,12 +19,12 @@
   "scripts": {
     "test": "yarn lint",
     "test:compiler": "jest ./tests",
-    "lint": "concurrently 'yarn lint:markdown' 'yarn lint:spelling' 'yarn lint:yaml' 'yarn lint:links'",
+    "lint": "concurrently 'yarn lint:links' 'yarn lint:markdown' 'yarn lint:spelling' 'yarn lint:yaml'",
```
Comment (elanelango) [inline package.json]: not a big deal, but this looks like a noop change?

Comment (cbetta) [reply package.json]: Agreed

File: content/pages/changelog/index.md (hunk 3 above)
Comment (cbetta) [reply content/pages/changelog/index.md]: Fixed

File: content/pages/changelog/index.md (hunk 2 above)
Comment (cbetta) [reply content/pages/changelog/index.md]: Fixed

**Comment by cbetta** [comment]
@elanelango I've reduced this PR to just the items relevant to DDOC-247

**Comment by cbetta** [comment]
Marked PR as `scheduled` as we're waiting for confirmation from @elanelango to confirm when this is going live. This might take longer than expected due to impact it might have on users.

**Pull request closed without merging**